### DISCUSSION TITLE
I broke PPInk

### DISCUSSION THREAD

**Discussion by fruvous** [pubpub-zz/ppInk#326] [General]
I don't know what I did but my pen no longer writes.  It previously worked and I'm using an Artisul D16.

I honestly don't think this is an issue with ppink but would appreciate any ideas or help.

Symptom:
PPInk toolbar comes up and I can write with my mouse but when I use the pen it moves around on the tablet but doesn't write but I can write with the mouse using left click.  The pen can stamp clipart but does not write.  

Writing within MS Whiteboard works but it's not using PPInk to work.

**Comment by pubpub-zz** [comment]
reinstall the drivers?

**Comment by fruvous** [comment]
I can stamp and the cursor moves around on the tablet so it's registering.  It's just not drawing :(

**Reply by pubpub-zz** [reply]
ppink draws based on pen pressure interface. drawing with click works too (either from mouse or pen).
If it used to work and do not work anymore, maybe some parameters of your tablet have been modified. reinstallation should reset these parameters.
